### ISSUE TITLE
deploy.sh: line 27: syntax error near unexpected token `<'

### ISSUE THREAD

**Issue by lostsheep007** [farski/serverless-transcribe#7]
@farski any help with this would be great. I followed the instructions, not sure why the script fails..

**Comment by farski** [comment]
@lostsheep007 What version of bash are you running?

**Comment by lostsheep007** [comment]
@farski Am now on 5.0.7 from 3.2! On a mac. Still getting that error though..

**Comment by farski** [comment]
@lostsheep007 Unfortunately I haven't been able to reproduce this error, so I don't have any ideas for how to resolve it

**Comment by lostsheep007** [comment]
Hi @Farski, so I worked it out yesterday. Windows Powershell, WSL or Mac Terminal with any version of bash installed all failed for me, then I used the actual bash shell in vs code and ran the script without a problem. Got there! Only other comment for relative beginners like me is to make sure aws configure is set for the right region on your machine before running the script and also that the stack naming needs to be lower case as it becomes part of the bucket name. Saves having to rerun..

Thanks for responding and sharing such a great piece of work!
